FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/SchedulerService.cs

```diff
@@ -40,13 +42,15 @@ public class SchedulerService : ISchedulerService
             IQueueClient queueClient,
             IMetadataStore metadataStore,
             IOptions<JobConfiguration> jobConfiguration,
+            IMetricsLogger metricsLogger,
             IDiagnosticLogger diagnosticLogger,
             ILogger<SchedulerService> logger)
         {
             _queueClient = EnsureArg.IsNotNull(queueClient, nameof(queueClient));
             EnsureArg.IsNotNull(jobConfiguration, nameof(jobConfiguration));
             _diagnosticLogger = EnsureArg.IsNotNull(diagnosticLogger, nameof(diagnosticLogger));
             _logger = EnsureArg.IsNotNull(logger, nameof(logger));
+            _metricsLogger = EnsureArg.IsNotNull(metricsLogger, nameof(metricsLogger));
```

Comment:
Do we need another error processor for scheduler service?

Reply:
Added a new processor for scheduler service.